src/python/zquantum/vqe/singlet_uccsd.py

```diff
@@ -18,16 +18,16 @@ class SingletUCCSDAnsatz(Ansatz):
 
     def __init__(
         self,
-        number_of_layers: int,
         number_of_spatial_orbitals: int,
         number_of_alpha_electrons: int,
+        number_of_layers: int = 1,
         transformation: str = "Jordan-Wigner",
     ):
         """ 
         Ansatz class representing Singlet UCCSD Ansatz.
 
         Args:
-            number_of_layers: number of layers of the ansatz.
+            number_of_layers: number of layers of the ansatz. Since it's a Singlet Ansatz, it can only be equal to 1.
```

Comment:
I feel like I'm nitpicking but I would rather write "The Singlet UCCSD ansatz implies number_of_layers = 1" just because the singlet part is not actually relevant to the number of layers, the UCCSD part is.

Reply:
Good point!